docker/start-app.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This configuration is a temporary workaround until we migrate to CCF 4.0.3, which supports
+# a cchost "config-timeout" option to accomplish a similar result. This logic is required when
+# the SCITT container is deployed in Kubernetes without any orchestration sidecars.
+
+WAIT_TIME_SEC=3
+TIMEOUT_SEC=300
+
+echo "Starting app"
+config_arg=$1
```

Comment:
I do not understand what this argument is, should you force it like:

```
config_arg=${1:?first argument must be the configuration path to pass to the cchost}
```

Reply:
Addressed